client/src/components/leftsidebar.js

```diff
@@ -38,21 +39,40 @@ class LeftSideBar extends React.Component {
           boxShadow: "1px 0px 6px 2px rgba(153,153,153,0.4)"
         }}
       >
-        <p
+        <div
```

Comment:
see line 31 - looks like a bad merge or other detritus to clean up?  (`scatterplotXXaccessor ....`)